_config.yml

```diff
@@ -64,14 +64,25 @@ exclude:
   - swagger/.cache
   - swagger/dist
   - swagger/node_modules
+  - .idea/
 
 
-paginate: 20
-paginate_path: "/news/page:num/"
+#paginate: 20
+#paginate_path: "/it/news/page:num/"
```

Comment:
Should we remove the commented lines before merging?

Reply:
Yes, this section has been moved inside the specific area

Reply:
furthermore it's been replaced by v2 new syntax